plugins/catalog-backend/scripts/mock-data

```diff
@@ -1,9 +1,16 @@
-#!/usr/bin/env sh
-curl \
---location \
---request POST 'localhost:7000/catalog/locations' \
---header 'Content-Type: application/json' \
---data-raw '{
-    "type": "github",
-    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/example-components.yaml"
-}'
+#!/usr/bin/env bash
+
+for URL in \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/artist-lookup-component.yaml' \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/playback-order-component.yaml' \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/podcast-api-component.yaml' \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/queue-proxy-component.yaml' \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/searcher-component.yaml' \
+  'https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/examples/shuffle-api-component.yaml' \
```

Comment:
Not just each filename here? 😅

Reply:
Yeah yeah :) Haha, sure, they will always be together. Fixed!

Reply:
👌